examples/index.html

```diff
@@ -62,7 +62,7 @@
       rel="stylesheet"
     />
     <script src="../node_modules/at.js/dist/js/jquery.atwho.min.js"></script>
-    <script src="data/emoji.js"></script>
+    <script src="../src/PublicLab.Emoji.js"></script>
```

Comment:
Hey @Shulammite-Aso , we should use `/dist` path here but can you confirm that the emojis are getting bundled correctly inside `/dist` folder?